README.md

```diff
@@ -1,18 +1,14 @@
 # unifi-acme.sh
 
-The `update-unifi-certificate.sh` script enables easy updating of the certificate used by [UniFi Controller](https://www.ubnt.com/enterprise/software). The script has been tested on Debian 8 "Jessie" with Unifi Controller installed via the official Debian repository.
+The `update-unifi-certificate.sh` script enables easy updating of the certificate used by [UniFi Controller](https://www.ubnt.com/enterprise/software). The script has been tested on Debian 8 "Jessie" with Unifi Controller installed via the official Debian repository and on a UniFi CloudKey on firmware version 0.7.3.
 
 ## How to use
 
 * Install [acme.sh](https://github.com/Neilpang/acme.sh) and follow the instructions to create a certificate for the domain you want to use to access UniFi Controller.
-* Update the `update-unifi-certificate.sh` script:
-  * Set the `DOMAIN` variable to the domain for which you previously generated a certificate using `acme.sh`.
-  * Set the `ACME_SH` variable to the location of the `acme.sh` installation.
-* Execute `update-unifi-certificate.sh` to update Unifi Controller.
-
-## TODO
-
-The `update-unifi-certificate.sh` script should be run automatically whenever a new certificate is generated via `acme.sh`. For now, you must manually execute `update-unifi-certificate.sh`.
+* Set the reload command to the `update-unifi-certificate.sh` script:
```

Comment:
Nice that we can rely on acme.sh for this. My first approach was just a cronjob.